CONTRIBUTING.md

```diff
@@ -129,40 +129,49 @@ It is used on bare systems to bootstrap the Swift toolchain (including SwiftPM),
 The bootstrap script requires having [CMake](https://cmake.org/) and [Ninja](https://ninja-build.org/) installed.
 Please refer to the [_Get Started_ guide](https://github.com/apple/swift/blob/main/docs/HowToGuides/GettingStarted.md#installing-dependencies) on the Swift project repository for installation instructions.
 
-1. Clone [llbuild](https://github.com/apple/swift-llbuild) beside the SwiftPM directory.
-
-```bash
-$> git clone https://github.com/apple/swift-llbuild llbuild
-```
-
-Note: Make sure the directory for llbuild is called "llbuild" and not
-    "swift-llbuild".
-
-2. Clone [Yams](https://github.com/jpsim/yams) beside the SwiftPM directory.
-
-```bash
-$> git clone https://github.com/jpsim/yams
-```
-
-3. Clone [swift-driver](https://github.com/apple/swift-driver) beside the SwiftPM directory.
-
-```bash
-$> git clone https://github.com/apple/swift-driver
-```
-
-4. Clone [swift-argument-parser](https://github.com/apple/swift-argument-parser) beside the SwiftPM directory and check out tag with the [latest version](https://github.com/apple/swift-argument-parser/tags).
-
-For example, if the latest tag is 0.3.1:
-```bash
-$> git clone https://github.com/apple/swift-argument-parser --branch 0.3.1
-```
-
-5. Clone [swift-crypto](https://github.com/apple/swift-crypto) beside the SwiftPM directory and check out tag with the [latest version](https://github.com/apple/swift-crypto/tags).
-
-For example, if the latest tag is 1.1.3:
-```bash
-$> git clone https://github.com/apple/swift-crypto --branch 1.1.3
-```
+Clone the following repositories beside the SwiftPM directory:
+
+1. [swift-driver]
+   ```sh
+   $> git clone https://github.com/apple/swift-driver
+   ```
+
+2. [swift-tools-support-core]
+   ```sh
+   $> git clone https://github.com/apple/swift-tools-support-core
+   ```
+
+3. [Yams]
+   ```sh
+   $> git clone https://github.com/jpsim/yams
```

Comment:
I think the dependency on Yams is also a version dependency (currently at 4.0.6, I think).

Reply:
Thanks for spotting this. I added `--branch 4.0.6` in 2c7df484d0ad9092fa56ad99cb8fdd05a8284a00.